contracts/modules/social_recovery/storage/GuardianStorage.sol

```diff
@@ -30,24 +30,22 @@ contract GuardianStorage is IGuardianStorage {
 
     /**
      * @dev Throws if the caller is not an enabled module.
-     * @param _wallet The target wallet.
      */
-    modifier onlyModule(address _wallet) {
+    modifier onlyModule() {
         // solhint-disable-next-line reason-string
-        require(ISafe(payable(_wallet)).isModuleEnabled(msg.sender), "GS: method only callable by an enabled module");
+        require(ISafe(payable(msg.sender)).isModuleEnabled(address(this)), "GS: method only callable by an enabled module");
         _;
     }
 
     /**
      * @dev Lets an authorised module add a guardian to a wallet and change the threshold.
-     * @param _wallet The target wallet.
      * @param _guardian The guardian to add.
      */
-    function addGuardianWithThreshold(address _wallet, address _guardian, uint256 _threshold) external onlyModule(_wallet) {
+    function addGuardianWithThreshold(address _guardian, uint256 _threshold) external onlyModule() {
```

Comment:
Specifically, from my comment in Slack, I would expect this to be:

```suggestion
    function addGuardianWithThreshold(address _wallet, address _guardian, uint256 _threshold) external authorized(_wallet) {
```

This ensures:
1. Consistency with the other `SocialRecoveryModule` methods
2. No breaking ABI change

Reply:
I removed this parameter because it is redundant. Also, `authorized` modifier is not needed. I have removed it here: https://github.com/5afe/CandideWalletContracts/pull/34/commits/78bb5d28578f19f41b767f6e03e180624572d6ac

If we want to keep `_wallet` as a parameter just for the consistency, I would revert this change.